content/articles/ms-ignite-2018.md

```diff
@@ -13,70 +13,42 @@ logo: /img/logo-sm.svg
-and Python. Azure Monitor
-[announcement](https://azure.microsoft.com/blog/new-full-stack-monitoring-capabilities-in-azure-monitor/)
-calls out that Python & Go apps monitoring are now supported by
-Application Insights thru Open Census and Local Forwarder.
+Included in the detailed list of ["new monitoring capabilities"](https://azure.microsoft.com/blog/new-full-stack-monitoring-capabilities-in-azure-monitor/) is the note that Azure Monitor for Applications now delivers better support for Go and Python applications. This is accomplished through the combination of OpenCensus and Microsoft's telemetry routing agent called Local Forwarder.
+
+Details are available in [Azure Monitor's distributed tracing documentation](https://docs.microsoft.com/azure/application-insights/distributed-tracing#enable-via-opencensus).
 
-You can find details documentation at
-[docs.microsoft.com](https://docs.microsoft.com/azure/application-insights/distributed-tracing#enable-via-opencensus).
+This announcement represents an important milestone: It not only enables support for Python and Go developers within Azure, but it also marks the first major contribution and collaboration between Microsoft and the OpenCensus community.
 
-This is an important milestone. It not only simply enables new apps
-support for Microsoft customers. It enables community to achieve more
-thru collaboration and combined scenarios.
+But this is not the only OpenCensus related news coming out of Microsoft.
 
 ### Call for standardization
 
-Open Census and Local Forwarder approach simplifies the design of
-monitoring system and opens door for new scenarios and innovations. All
-those scenarios are only possible with the standardized and unified
-implementation for metrics and distributed tracing data collection.
+OpenCensus simplifies the design of monitoring systems and opens the door for expanded polyglot scenarios and innovation. The power of these scenarios is of course amplified with support from the open source community, but also through efforts to bring unification and standardization across observability vendors.
 
-As a fundamental principle of Open Census we strive for consistency in
-data collection in SDKs thru specifications and test cases across
-languages. We build test cases for [propagation
-protocols](https://github.com/w3c/distributed-tracing) and data
-collection specifications.
+To that end, it is a fundamental principle of OpenCensus to strive for consistency across our libraries thru specifications. Microsoft has contributed to those efforts by contributing a test harness to help ensure compliance with the [Trace Context](https://github.com/w3c/distributed-tracing) propagation format, as well as Trace Context implementation to several OpenCensus language libraries. Microsoft will continue to champion open standards through participation in the W3C Distributed Tracing Working Group and W3C TPAC event in France this month.
```

Comment:
There is more than W3C I alluded to. Like this one: https://github.com/census-instrumentation/opencensus-specs/pull/183

Reply:
standard on data collection unification

Reply:
I've called #183 the "test harness". Would you like me to link to it, or are you saying I've called it the wrong thing?

Also, I'm not sure what *standard on data collection unification* means. Could you elaborate please?